### PR TITLE
Adding parameters to authenticate with nuget

### PR DESCRIPTION
**Pull request by JosephDSchwartz**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -11,18 +11,29 @@ inputs:
     required: true
   install-mock-db-objects:
     description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur.
```
Comment (hpractv) [inline action.yml:13]: We might want to call out specifically that you're expecting a `true` or `false` string value.

File: action.yml [hunk 2]
```diff
@@ -11,18 +11,29 @@ inputs:
...
+  nuget-password:
+    description: The password for the user to authenticate with the nuget feed. This should be set if install-mock-db-objects is true.
+    required: false
   incremental:
     description: Specifies whether to drop and recreate the database before building, or apply to the current database.
-    default: false
+    required: false
+    default: 'false'
   run-tests:
     description: Specifies whether or not to run tests.
```
Comment (hpractv) [inline action.yml:30]: Same thing here `true` or `false` expected value.

**Review by hpractv — CHANGES_REQUESTED** [review]

File: action.yml [hunk 3]
```diff
@@ -10,19 +10,30 @@ inputs:
     description: The name of the database to bubild.
     required: true
   install-mock-db-objects:
-    description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur.
-    default: false
+    description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur. The accepted values are true and false.
```
Comment (hpractv) [inline action.yml:13]: This is only a suggestion, but can we word it like this:
_The expected value is true or false._

That way it's clear a single value is expected.

Comment (JosephDSchwartz) [reply action.yml]: I'm cool with that!

**Review by danielle-casella-adams — APPROVED** [review]
This looks good, assuming it is all working. :)

**Review by hpractv — APPROVED** [review]
👍🏽

**Pull request merged**